src/metabase/api/collection.clj

```diff
@@ -92,18 +95,17 @@
                               (db/reducible-query {:select    [:collection_id :dataset]
                                                    :modifiers [:distinct]
                                                    :from      [:report_card]
-                                                   :where     [:= :archived false]}))]
-    (collection/collections->tree
-     coll-type-ids
-     (db/select Collection
-                {:where [:and
-                         (when exclude-archived
-                           [:= :archived false])
-                         [:= :namespace namespace]
-                         (collection/visible-collection-ids->honeysql-filter-clause
-                          :id
-                          (collection/permissions-set->visible-collection-ids @api/*current-user-permissions-set*))]}))))
-
+                                                   :where     [:= :archived false]}))
+        collections   (->> (db/select Collection
+                                      {:where [:and
+                                               (when exclude-archived
+                                                 [:= :archived false])
+                                               [:= :namespace namespace]
+                                               (collection/visible-collection-ids->honeysql-filter-clause
+                                                 :id
+                                                 (collection/permissions-set->visible-collection-ids @api/*current-user-permissions-set*))]})
```

Comment:
nit: function args on a new line should be at the [same indentation level as the function name](https://guide.clojure.style/#one-space-indent)